forest/src/daemon.rs

```diff
@@ -154,6 +157,26 @@ pub(super) async fn start(config: Config) {
 
     sync_from_snapshot(&config, &state_manager).await;
 
+    // Terminate if no snapshot is provided and DB isn't recent enough
+    match chain_store.heaviest_tipset().await {
+        None => {
+            cli_error_and_die(
+                "Forest cannot sync without a snapshot. Download a snapshot from a trusted source and import with --import-snapshot=[file]",
```

Comment:
This is great but may be a bit puzzling for new folks. They most likely don't know where is the trusted source. How about we add a section in README `Trusted snapshot sources` to make their life easier?

Reply:
Once we're hosting our own trusted snapshots, we can offer to automatically download them.